src/v/redpanda/admin/api-doc/redpanda_services_restart.json

```diff
@@ -0,0 +1,28 @@
+"/v1/redpanda-services/restart": {
+  "put": {
+    "summary": "Restart a redpanda service.",
+    "operationId": "redpanda_services_restart",
+    "parameters": [
+      {
+        "name": "service",
+        "in": "query",
+        "required": true,
+        "type": "string"
+      }
+    ],
+    "produces": [
+      "application/json"
+    ],
+    "responses": {
+      "200": {
+        "description": "Restart success"
+      },
+      "404": {
+        "description": "Service not found"
+      },
```

Comment:
I think 400 is also needed here for bad_request?

Reply:
Correct.